source/core/oxview.php

```diff
@@ -1017,4 +1017,60 @@ public function getSidForWidget()
 
         return $sRet;
     }
+
+    /**
+     * persparam visible, or as hidden field (in checkout)
+     * 
+     * @param $s_paramKey string
+     *
+     * @return bool
+     */
+    public function showPersParam($s_paramKey) {
+
+        $bRet = false;
+
+        if ($s_paramKey == "details")
+            $bRet = true;
+
+        return $bRet;
+    }
+
+    /**
+     * return label text for persparam (getter for text)
+     * 
+     * @param $sOxMultilangIdent
+     *
+     * @return string
+     */
+    public function getPersParamText($sOxMultilangIdent) {
+
+        if ($sOxMultilangIdent == "details") {
+            // details has different multilang-idents
+            if (isAdmin())
+                $sOxMultilangIdent = "ORDER_PACKAGE_DETAILS";
+            else
+                $sOxMultilangIdent = "LABEL";
+        }
+
+        return oxRegistry::getLang()->translateString($sOxMultilangIdent);
+    }
+
+    /**
+     * return value for persparam (getter for value)
+     * 
+     * @param $s_paramKey
+     * @param $s_paramValue
+     *
+     * @return null|other
+     */
+    public function getPersParamValue($s_paramKey, $s_paramValue) {
+
+        $_ret = null;
+
+        if ($s_paramKey == "details")
```

Comment:
Your right.
My thought was, the function is intended to be overloaded by modules... and the module returns "custom" values or no value or whatever ...
But at "default", it should return every persparam value.